### PR TITLE
Add missing German translation

### PR DESCRIPTION
**Pull request by maurin88**
Added missing german translation for empty.buttons.reset_column_searches.label

### CODE REVIEW

File: packages/tables/resources/lang/de/table.php [hunk 1]
```diff
@@ -95,6 +95,15 @@
 
     'empty' => [
         'heading' => 'Keine Datensätze gefunden',
```
Comment (zepfietje) [inline packages/tables/resources/lang/de/table.php:98]:
```suggestion

        'heading' => 'Keine Datensätze gefunden',
```

**Comment by zepfietje** [comment]
Thanks!

**Comment by maurin88** [comment]
Thank you very much. If you ever need additional help with german translations, give me a notice. Filament rocks !!!!

**Comment by zepfietje** [comment]
Have you joined our [Discord community](https://filamentphp.com/discord) yet?
We have a channel for dedicated translation managers. 😄

**Comment by maurin88** [comment]
Thanks for the advise, I will do this.

**Pull request merged**